Sources/Sentry/SentryOptions.m

```diff
@@ -25,14 +33,16 @@ - (instancetype)init
         self.maxBreadcrumbs = defaultMaxBreadcrumbs;
         self.maxCacheItems = 30;
         self.integrations = SentryOptions.defaultIntegrations;
-        self.sampleRate = @1;
+        _defaultSampleRate = @1;
```

Comment:
Shouldn't this just be a const defined inline somewhere?

Reply:
Inline not, I also need it for the setters. Unfortunately, you can't use NSNumber for constants. That's a drawback of Objective-C. We could do some weird stuff as described here https://stackoverflow.com/questions/12268904/nsnumber-constants-in-obj-c, but instead, I prefer to use private properties instead.